gcc/rust/rust-diagnostics.cc

```diff
@@ -197,11 +197,6 @@ rust_error_at (const RichLocation location, const char *fmt, ...)
   va_end (ap);
 }
 
-// rust_debug uses normal printf formatting, not GCC diagnostic formatting.
-
-void
-rust_debug (const Location location, const char *fmt, ...) ATTRIBUTE_PRINTF_2;
-
```

Comment:
Unless there was a reason to move it, I suggest you leave this prototype in its original place; it applies to the non-templated `rust_debug` function.

Reply:
The reason was in #464, I assumed this prototype needed to be available too in case there was a need for rust_debug with a given location instead of an empty one

Reply:
Right.  I failed to see that it moved from the `*.cc` into the `*.h` file.  ;-)